src/zapv2/__init__.py

```diff
@@ -138,14 +139,23 @@ def _request(self, url, get=None):
            - `url`: the url to GET at.
            - `get`: the dictionary to turn into GET variables.
         """
-        return json.loads(self._request_api(url, get))
+        return self._request_api(url, get).json()
+
+    def _request_value(self, endpoint_url, **query_params):
+        """
```

Comment:
I introduced this method as an example of refactoring -> I can refactor the rest of the code to use this method if you like it.